irohad/main/irohad.cpp

```diff
@@ -154,6 +162,30 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+  boost::optional<std::vector<std::unique_ptr<shared_model::interface::Peer>>>
+      peers = boost::make_optional<
+          std::vector<std::unique_ptr<shared_model::interface::Peer>>>({});
+  if (not FLAGS_peers.empty()) {
+    iroha::main::PeersFileReaderImpl file_reader;
```

Comment:
so this could bee 
```suggestion
    iroha::main::PeersFileReaderImpl file_reader(<make_peers_factory...>);
    auto opt_override_peers = file_reader.readPeers(FLAGS_peers);
```

Reply:
:bee: